burton/parser/resx.py

```diff
@@ -136,6 +138,81 @@ def _rewrite_mapping(key, value, node):
 
             file.close()
 
+            if(proj_file != None):
```

Comment:
I thought I saw that the default is an empty string (if I read that write). Won't that fall through here in that case?

Reply:
Its kind of a misnomer. The config parsing stuff requires the value to exist in the config file for each section (or be in the defaults section), and each section (including the defaults section) must set its value (you can override this so that it doesn't require a value, but burton doesn't).

So the expectation is they would put in the config file:
project_path= which i believe results in None. But now i'm not completely certain so I'll research some more

Reply:
looks like you're right. although it is documented that you can set it to be proj_path= without a value in the config file, its not actually working in practice. so i changed it to compare to the string None, which actually appears to get lower cased.